app/views/landing/_product.html.erb

```diff
@@ -1,10 +1,15 @@
-<div class= "col s12 m6 l3">
-	<%= link_to product_path(product) do %>
-		<div class="row card home-card">
-		  <div class="card-image"><%= image_tag product.product_image_links.first.link_name %></div>
-		</div>
-		<div class="card-desc center">
-	      <span><%= product.name %></span><br /><span class="price">&#x20A6;<%= product.price %></span>
-	    </div>
-	<% end %>
+<div class="col s12 m6 l3">
+  <%= link_to product_path(product) do %>	  
+   	<div class="card"> 
+  	  <div class="card-image">
+        <%= image_tag product.product_image_links.first.link_name %>
+      </div>
+      <div class="card-content center">
+        <span><%= product.name %></span><br>
+        <span class="price">&#x20A6;<%= product.price %></span>
+      </div>
+  	</div>	
```

Comment:
It looks properly indented on my system, I'm surprised it's showing this way here. I'll correct it.